README.md

```diff
@@ -124,7 +124,7 @@ Use command line under the root directory of `llvm-ventus`
 Linking CRT and libclc
 All the libclc workitem is included in `riscv32clc.o`
 ```sh
-./install/bin/ld.lld -o vecadd.riscv -T utils/ldscripts/ventus/elf32lriscv.ld vecadd.o ./install/lib/crt0.o ./install/lib/riscv32clc.o -L./install/lib -lworkitem --gc-sections
+./install/bin/ld.lld -o vecadd.riscv -T utils/ldscripts/ventus/elf32lriscv.ld vecadd.o ./install/lib/crt0.o ./install/lib/riscv32clc.o ./install/lib/workitem.o --gc-sections
```

Comment:
Would it be better to make it a dynamic link directly? In this way, there is no need to consider the issue of gc-section. The codesize will become smaller, but there may be a 1%-2% loss in performance.

Reply:
No dynamic link since dynamic library require operating system